python/taichi/lang/expr.py

```diff
@@ -67,9 +67,6 @@ def __getitem__(self, key):
     def loop_range(self):
         return self
 
-    def serialize(self):
-        return self.ptr.serialize()
```

Comment:
I'm not familiar with this method yet, why do we want to remove it? Expr serialization still sounds useful to me.

Reply:
In fact it's never used in Python-side... it's only used in `ir_printer.cpp`.

Reply:
I see, that makes sense then!